Automattic/wp-calypso
# Stats: Adding header and footer to Stats List component

#### Proposed Changes

* This PR adds a header and footer to the Stats list that uses the Horizontal Bar component (not used outside of Storybook yet)
* Adding new wrapping component - `StatsCard`
* Combining variables used by the module into one file
* updating Storybook stories and adding one new

#### Testing Instructions

* checkout this branch and open Storybook
* new stories can be found under `Horizontal bar list` and are called `Default` and `Default with Footer`

The updated component should look like this:
![Screenshot 2022-10-21 at 5 25 29 PM](https://user-images.githubusercontent.com/112354940/197111758-fb4e024c-66df-4796-8638-33ea731847b7.png)


#### Pre-merge Checklist






Related to #68981
